Minimal CSS Question Components

rarchitgupta opened the pull request.

rarchitgupta committed:
Minimal CSS Question Components

arnavb reviewed (changes requested):
Make sure to use bulma styling for containers, etc. Position related styling should be minimal, the bulma defaults are good to follow. Be a little more consistent with classnames (use dash case everywhere, I notice camelCase in some places). 

Also, could you make an example component with some sample questions as a page? That way I can see what it looks like in practice.

rarchitgupta commented:
Alright I'll see to it

rarchitgupta committed:
Added quiz page for view, changed classnames

rarchitgupta commented:
I added a quiz page you can visit by manually adding "/quiz" into the URL. Didn't mess with the navbar. There's an array of question objects you can change to view the questions. Also used Bulma and changed classnames

arnavb commented on .prettierrc:
```diff
@@ -0,0 +1 @@
+{}
```
Delete this file.

arnavb commented on src/pages/quiz.css:
```diff
@@ -0,0 +1,3 @@
+body {
```
Don't set global styles, this will propagate across all pages since we're dealing with a single page app.

arnavb commented on src/pages/quiz.js:
```diff
@@ -0,0 +1,46 @@
+
+export default function Quiz() {
+  return (
+    <div className="container">
```
Regarding answers, I think radio choices stacked vertically would be better. In addition, I think you should use the Bulma form controls for input related things. There's lots of quiz apps for inspiration, such as:

![image](https://user-images.githubusercontent.com/30810402/93531882-f182e580-f90d-11ea-8984-cc42a94408f2.png)

or something simpler:

![image](https://user-images.githubusercontent.com/30810402/93531916-fe9fd480-f90d-11ea-89db-017b29cf74b6.png)

rarchitgupta commented:
Yeah sorry I didn't mean to push the prettier config. I'll look into the rest too.

rarchitgupta committed:
Vertically Aligned Answer Options, fixed form controls

rarchitgupta commented:
I can't figure out what's causing the blank space below the last question. Rest everything I think I've taken care of

lint-action committed:
Fix code style issues with Prettier

arnavb committed:
Make some styling changes

arnavb committed:
Minor color change
